scripts/release/create-release.hs

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env cabal 
+{- project:
+  with-compiler: ghc-9.4
+-}
+{- cabal:
+  build-depends:
+    base,
+    commonmark,
+    containers,
+    filepath,
+    foldl,
+    text,
+    turtle ^>=1.6.0,
+-}
+{-# OPTIONS_GHC -Wall -Wextra #-}
+{-# LANGUAGE GHC2021 #-}
+{-# LANGUAGE BlockArguments, LambdaCase, OverloadedStrings, ScopedTypeVariables #-}
```

Comment:
I think this didn't pass through stylish Haskell. Perhaps we can format it?

Reply:
done! should I add this to the `check-stylish` script too

Reply:
I would do so, yeah